### PR TITLE
added the solution for #3 in go

### PR DESCRIPTION
**Pull request by surasnayak**
Link: https://ideone.com/I91we1

### CODE REVIEW

**Review by gauthamzz — CHANGES_REQUESTED** [review]
Need as a blog post

**Comment by surasnayak** [comment]
can you please show me how to do it so that I can proceed further

**Comment by surasnayak** [comment]
@gauthamzz Can you help please me out by showing how to make a blog post ?
So that I can proceed further smoothly : )

**Comment by surasnayak** [comment]
@gauthamzz I have resolved the problem and added blog posts for the solution of DAY3 and DAY4 instead of the program files.
Please check.

**Review by gauthamzz — CHANGES_REQUESTED** [review]
The way for highlighting code is not applied

**Comment by surasnayak** [comment]
Fixed the highlighting part. Please check
Note: Adding SS for reference
screenshot from 2018-01-13 12-15-18

File: _posts/2018-01-13-DAY-3-solution-in-go.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
```
Comment (mubaris) [inline _posts/2018-01-13-DAY-3-solution-in-go.md]: Add title, author, layout and date

[Read this](https://jekyllrb.com/docs/posts/)

File: _posts/2018-01-13-DAY-3-solution-in-go.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+published: true
```
Comment (mubaris) [inline _posts/2018-01-13-DAY-3-solution-in-go.md:2]: Add title, author, layout and date

[Read this](https://jekyllrb.com/docs/posts/)

**Review by mubaris — CHANGES_REQUESTED** [review]

File: _posts/2018-01-13-DAY-4-solution-in-go.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
+---
+published: true
```
Comment (mubaris) [inline _posts/2018-01-13-DAY-4-solution-in-go.md:2]: Add title, author, layout and date

[Read this](https://jekyllrb.com/docs/posts/)

**Review by mubaris — CHANGES_REQUESTED** [review]

File: _posts/2018-01-13-DAY-3-solution-in-go.md (hunk 2 above)
Comment (surasnayak) [reply _posts/2018-01-13-DAY-3-solution-in-go.md:2]: @mubaris what should be the title? and what do you mean by layout?

Comment (mubaris) [reply _posts/2018-01-13-DAY-3-solution-in-go.md:2]: Put title something meaningful, not "Day 3 Solution". Create a title from the question

and layout is post

```
layout: post
```

**Review by gauthamzz — APPROVED** [review]

**Comment by namantw** [comment]
@gauthamzz is this a mistake? I solved the question 2 and 3 first, shouldn't I be the one adding blog post for question 2 and 3. I was working on the theme issue so it was on hold but I  had given the IDEONE links in pull request no. 7. I think this is kind of unfair. Please clarify :sad:

**Pull request merged**